pytest/lib/branches.py

```diff
@@ -63,7 +63,7 @@ class Executables(typing.NamedTuple):
     def node_config(self) -> typing.Dict[str, typing.Any]:
         return {
             'local': True,
-            'neard_root': self.root,
+            'near_root': self.root,
```

Comment:
Can you confirm that no other codepath relies on this? I guess just schedule a full nayduck run and see that it doesn't break any tests.

Reply:
there are literally 0 occurrences of `neard_root` :) will do, anyway